web/src/components/config/useConfigForm.ts

```
@@ -0,0 +1,287 @@
+  return JSON.parse(JSON.stringify(obj));
+}
+
+/** Recursively scan for MASKED strings and collect their dotted paths. */
+function scanMasked(obj: unknown, prefix: string, out: Set<string>) {
+  if (obj === null || obj === undefined) return;
+  if (typeof obj === 'string' && obj === MASKED) {
+    out.add(prefix);
+    return;
+  }
+  if (Array.isArray(obj)) {
+    obj.forEach((item, i) => scanMasked(item, `${prefix}.${i}`, out));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)